src/adv.py

```diff
@@ -39,14 +41,60 @@
 #
 
```

Comment:
Great work Jesse! You can certainly add more features to this and make the game your own! :grin: